### PR TITLE
- added update category method

### PR DESCRIPTION
**Pull request by sshitov**
- rewrote restController methods

### CODE REVIEW

File: src/main/java/com/litequizapp/server/CategoryRestController.java [hunk 1]
```diff
@@ -20,34 +24,37 @@ public CategoryRestController(CategoryRestService restService) {
...
-  @PostMapping("/delete")
+  @PostMapping(value = "/{id}")
+  public void updateCategoryRequest(@PathVariable String id, @RequestBody String title) {
+    long categoryId = Long.parseLong(id);
+    restService.updateCategory(categoryId, title);
+
+  }
+
+  @DeleteMapping
   public void deleteCategoryByIdRequest(@RequestBody String id) {
```
Comment (malchun) [inline src/main/java/com/litequizapp/server/CategoryRestController.java]: It's not intuitive that we had all previous operation by id with id as path variable, and delete with id as body.

File: src/main/java/com/litequizapp/server/CategoryEntity.java [hunk 2]
```diff
@@ -39,6 +39,10 @@ public String getTitle() {
     return title;
   }
 
+  public void setTitle(String title){
```
Comment (malchun) [inline src/main/java/com/litequizapp/server/CategoryEntity.java]: I'd suggest to rewrite this with lombook.

File: src/main/java/com/litequizapp/server/LiteQuizApplication.java [hunk 3]
```diff
@@ -1,50 +1,17 @@
...
+import org.slf4j.LoggerFactory;*/
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class LiteQuizApplication {
 
-  private static final Logger log = LoggerFactory.getLogger(LiteQuizApplication.class);
+/*  private static final Logger log = LoggerFactory.getLogger(LiteQuizApplication.class);*/
```
Comment (malchun) [inline src/main/java/com/litequizapp/server/LiteQuizApplication.java]: In general it's bad idea to keep commented code in your repo.You can find this snapshot in git history if you need it.

File: src/main/resources/database/initDB.sql [hunk 4]
```diff
@@ -1,9 +1,8 @@
...
 CREATE TABLE IF NOT EXISTS categories
 (
     id     BIGSERIAL PRIMARY KEY NOT NULL,
     title  VARCHAR(100) NOT NULL
-);
\ No newline at end of file
+);
+
+CREATE SEQUENCE hibernate_sequence START 1;
+SELECT setval('hibernate_sequence', (SELECT max(id) FROM public.categories));
```
Comment (malchun) [inline src/main/resources/database/initDB.sql]: If it fixes the problem with liquibase i'd suggest to remove commented line in compose file.

**Review by malchun — CHANGES_REQUESTED** [review]
Got some suggestions for you.

File: src/main/resources/database/initDB.sql (hunk 4 above)
Comment (sshitov) [reply src/main/resources/database/initDB.sql]: done

**Pull request merged**